spec/models/git_repository_spec.rb

```diff
@@ -12,14 +12,12 @@
   end
 
   it "default dirname" do
-    repo = FactoryBot.create(:git_repository,
-                              :url => "http://www.example.com/repos/manageiq")
+    repo = FactoryBot.create(:git_repository, :url => "http://www.example.com/repos/manageiq")
```

Comment:
Oh man, I would have probably been a bit triggered if you had left that `:url` indented as it was previously... 😅